libs/execution/src/lib/constraints/constraint-executor-registry.ts

```diff
@@ -46,7 +46,8 @@ export function createConstraintExecutor(
   constraint: ConstraintDefinition,
 ): ConstraintExecutor {
   if (isTypedConstraintDefinition(constraint)) {
-    const constraintType = constraint.type.name;
+    const constraintType = constraint.type.ref?.name;
+    assert(constraintType !== undefined);
```

Comment:
We can't keep doing asserts without error messages to handle errors. It has to have an error message, otherwise you just get a 'XYZ evaluated to a falsy value'.

Reply:
True, added in [6112703](https://github.com/jvalue/jayvee/pull/444/commits/611270315903f34152152fdbfaf27c2e4b6e3f55)